### DISCUSSION TITLE
Updating to org 9.7 in spacemacs

### DISCUSSION THREAD

**Discussion by RaghuNaik** [syl20bnr/spacemacs#16445] [General]
What is the right way to update org to 9.7 in spacemacs? Thanks.

**Comment by RaghuNaik** [comment]
Just installed org 9.7.3 through the package manager.  It seems to have worked. Wondering if that is the right way. Thanks again.

**Reply by smile13241324** [reply]
Basically we can update once org 9.7 is available via elpa. If this takes too long installing it from source or via the emacs package manager is the way to go.
